src/Pos/ReportServer/Server.hs

```diff
@@ -124,7 +124,7 @@ reportApp ServerContext{..} req respond =
               Left e -> do
                   let ex = displayException e
                   hPutStrLn stderr ("An exception occurred: " <> ex)
```

Comment:
Shouldn't we filter out the basic authentication field from the exception that occured?

We arguably don't want the ZenDesk token in Papertrail as well.

Maybe we should also audit what other exceptions could pop up with secrets here.

Reply:
Quoting Domen:
> yes but that's limited impact, meanwhile our report server is offline
> it's very tricky to fix this for papertrail
> because the error comes from wai
> another reason to  use a proper error reporting software
> they allow filtering things like this on their side which is easier than in code

Reply:
@domenkozar, I wouldn't block review on this, but I think at least making an issue would make sense here. I'd even say it to be of somewhat higher priority.

Reply:
Good call - CSL-2374